crates/sparrow-session/src/partitioned.rs

```diff
@@ -1,5 +1,5 @@
 mod session;
 pub use session::*;
 
-#[cfg(tests)]
```

Comment:
The configuration is `#[cfg(test)]`, and the standard module name is `tests`. Since nothing set the `tests` configuration, this was never being included.